frontend/src/lib/candidate/components/input/SingleChoiceInput.svelte

```diff
@@ -49,18 +51,16 @@ A component for a single choice question that can be answered.
   <p slot="header" class="small-label mx-6 my-0 p-0">
     {headerText}
   </p>
-  <Field id={question.id} label={question.text}>
+  <Field id={questionId} label={headerText}>
     <InputContainer {locked}>
       <select
         disabled={locked}
-        id={question.id}
-        data-testid={question.id}
+        id={questionId}
+        data-testid={questionId}
```

Comment:
Can we remove this and update the `register.spec.ts` to just use `getById` instead? Could you also do the same thing for the submit button in `profile/+page.svelte`.